frontend/src/components/Row.tsx

```diff
@@ -99,12 +99,14 @@ export const Row = ({
           </p>
         </div>
         <div className="col-1 star-container">
+	<button type="button" className="star-button" onClick={() => onToggleFav(topic)}>
           <img
             src={isFav ? fillStar : star}
-            onClick={() => onToggleFav(topic)}
             className="star"
             role="button"
+            alt="Make Favourite"
```

Comment:
This alt text should change depending on if the star is filled or not, i.e. if the issue is a favorite or not: 
```suggestion
            alt={isFav ? "Remove from favorites" : "Make favorite"}
```
(and we have decided to always use the American way of spelling, after some problems due to different variants of "color" and "colour" in a different project... :) )